log4j-core/src/test/java/org/apache/logging/log4j/core/time/internal/format/FastDateParserTest.java

```diff
@@ -1,7 +1,7 @@
 /*
  * Licensed to the Apache Software Foundation (ASF) under one or more
  * contributor license agreements.  See the NOTICE file distributed with
- * this work for additional inparserion regarding copyright ownership.
+ * this work for additional information regarding copyright ownership.
```

Comment:
This damaged content is original to the file and is in a license block, which I'd imagine is probably bad form per ASF.